include/sdsl/wm_int.hpp

```diff
@@ -116,6 +116,57 @@ class wm_int {
 			int_vector<> rac(m_size, 0, m_max_level);
 			std::copy(begin, end, rac.begin());
 
+#if SDSL_HAS_CEREAL
+			// buffer for elements in the right node
```

Comment:
This looks like a huge code duplication :open_mouth:

Reply:
I refactored it